App/screens/CredentialOffer.tsx

```diff
@@ -31,123 +34,187 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
     flexDirection: 'column',
-    backgroundColor: CredentialOfferTheme.background,
+    backgroundColor: ColorPallet.brand.primaryBackground,
+  },
+  listItem: {
+    paddingHorizontal: 25,
+    paddingTop: 16,
+    backgroundColor: ColorPallet.brand.secondaryBackground,
+  },
+  label: {
+    ...TextTheme.label,
+  },
+  textContainer: {
+    justifyContent: 'space-between',
+    minHeight: TextTheme.normal.fontSize,
+    paddingVertical: 15,
+    height: 90,
+  },
+  text: {
+    ...TextTheme.normal,
   },
 })
 
 const CredentialOffer: React.FC<CredentialOfferProps> = ({ navigation, route }) => {
+  if (!route || !route.params) {
+    throw new Error('CredentialOffer route prams were not set properly')
+  }
+
+  const { credentialId } = route.params
   const { agent } = useAgent()
   const { t } = useTranslation()
+  const [, dispatch] = useContext(Context)
   const [buttonsVisible, setButtonsVisible] = useState(true)
   const [pendingModalVisible, setPendingModalVisible] = useState(false)
   const [successModalVisible, setSuccessModalVisible] = useState(false)
   const [declinedModalVisible, setDeclinedModalVisible] = useState(false)
+  const dateFormatOptions = { year: 'numeric', month: 'short', day: 'numeric' }
```

Comment:
Luxon is one of the packages currently used to parse dates. Not sure if we want to keep using that or refactor it out. There's a formatted string available in the `constants` module if using that package.

Reply:
Should we then remove it in favour of JS `Date`. I don't see value in maintaining two ways of doing something.

Reply:
I think so.